src/cmd_vel_mux.cpp

```diff
@@ -36,31 +36,56 @@ namespace cmd_vel_mux
 
 CmdVelMux::CmdVelMux(rclcpp::NodeOptions options) : rclcpp::Node("cmd_vel_mux", options.allow_undeclared_parameters(true).automatically_declare_parameters_from_overrides(true)), allowed_(VACANT)
 {
-  std::vector<std::string> names;
-  if (!get_parameter("subscribers.name", names))
+  std::map<std::string, rclcpp::Parameter> subs_parameters;
+  std::map<std::string, std::string> topics;
+  std::map<std::string, double> timeouts;
+  std::map<std::string, int64_t> priorities;
+  std::map<std::string, std::string> short_descs;
+
+  if (!get_parameters("", subs_parameters) || subs_parameters.size() <= 1)
   {
     RCLCPP_WARN(get_logger(), "No subscribers configured!");
   }
-
-  std::vector<std::string> topics;
-  get_parameter("subscribers.topic", topics);
-
-  std::vector<double> timeouts;
-  get_parameter("subscribers.timeout", timeouts);
-
-  std::vector<int64_t> priorities;
-  get_parameter("subscribers.priority", priorities);
-
-  std::vector<std::string> short_descs;
-  get_parameter("subscribers.short_desc", short_descs);
-
-  if (names.size() != topics.size() || names.size() != timeouts.size() ||
-      names.size() != priorities.size() || names.size() != short_descs.size())
+  else
   {
-    throw std::runtime_error("Number of names, topics, timeouts, priorities, and short_descs must be the same");
+    for(auto& x : subs_parameters)
```

Comment:
Nit: please don't use auto here.  It's hard to for later readers to figure out what the type is.  Also, space after `for`.